rfcs/tag-component.md

```diff
@@ -0,0 +1,53 @@
+# Tag Component
+
+## Exports
+
+* `Tag`
+* `StyledRoot`
+* `StyledAction`
+
+## `Tag` API
+
+* `color: STYLE.blue(theme.colors.primary470) | STYLE.orange(theme.colors.warning400) | STYLE.green(theme.colors.positive400) | STYLE.red(theme.colors.negative400)`:
+  the color theme to be applied to a Tag. Default is `STYLE.blue`.
+* `children: React$Node`:
+  Component or String value for label of tag. Default is empty string
+* `disabled: boolean`:
+  Disable control from being changed
+* `overrides: {Root: (props: {[string]: any}) => React$Node, Action: (props: {[string]: any}) => React$Node}`
+  * Custom `Action` button.
+  * `Root` wrapper element for the whole tag control to apply styles
+* `onActionClick: func(evt => void)`:
+  handler for events on Action button element (default is Cross).
+
+## Usage
```

Comment:
Looks like we put `Usage` before the exports and API for docs reasons

Reply:
ok, will fix this